Makefile

```diff
@@ -402,6 +402,41 @@ cluster-logging-operator-install:
 cluster-logging-operator-uninstall:
 	olm_deploy/scripts/operator-uninstall.sh
 
-.PHONY: gen-dockerfiles
-gen-dockerfiles:
-	./hack/generate-dockerfile-from-midstream > Dockerfile
+REPOSITORY_BASE ?= $(error REPOSITORY_BASE needs to be set for the olm- targets to work)
```

Comment:
Can we use REGISTRY_BASE please? That's what loki does and I've followed that convention elsewhere. Would be nice to set just one env.var :)

Reply:
I retract this comment :)

Reply:
Too late. I already made it possible to use both variants (with the preference being `REGISTRY_BASE`). :grin: